src/cljs/main/broadfcui/common/components.cljs

```diff
@@ -678,3 +678,34 @@
 (defn create-error-message [thing]
   (when (renderable? thing)
     #(push-error thing)))
+
+(react/defc TagAutocomplete
+  {:get-tags
+   (fn [{:keys [refs]}]
+     (flatten (js->clj (.tagsinput (js/$ (@refs "input-element")) "items"))))
+   :component-did-mount
+   (fn [{:keys [refs]}]
+     (let [suggestion-engine (Bloodhound.
+                               (clj->js {:datumTokenizer whitespace-tokenizer
+                                         :queryTokenizer whitespace-tokenizer
+                                         :remote {:url (str (config/api-url-root) "/api/workspaces/tags/autocomplete/")
```

Comment:
Should this be a method in endpoints.cljs or are we getting away from centralizing urls there?